drawer/src/space/earlygrey/shapedrawer/GraphDrawer.java

```diff
@@ -0,0 +1,390 @@
+package space.earlygrey.shapedrawer;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.math.Interpolation;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.utils.FloatArray;
+
+/**
+ * Contains functions for drawing graphs of {@link Interpolation} instances. Use the default interpolations or create
+ * your own! GraphDrawer assumes a single, continuous graph formula is provided.
+ *
+ * @author Raymond "Raeleus" Buckley
+ */
+public class GraphDrawer {
+    /**
+     * The {@link ShapeDrawer} used to render the graph path.
+     */
+    public ShapeDrawer shapeDrawer;
```

Comment:
Since you have getters and setters for all of these fields, can we make them private? I think it would make changing internal implementations in the future easier, or eg adding side effects to the getters/setters and being sure everyone is using them.